dcicutils/secrets_utils.py

```diff
@@ -56,26 +88,22 @@ def get_identity_secrets(identity_kind: str = GLOBAL_APPLICATION_CONFIGURATION)
             SecretId=secret_name
         )
     except ClientError as e:  # leaving some useful debug info to help narrow issues
-        if e.response['Error']['Code'] in [
-            'DecryptionFailureException'  # SM can't decrypt the protected secret text using the provided KMS key.
-            'InternalServiceErrorException',  # An error occurred on the server side.
-            'InvalidParameterException',  # You provided an invalid value for a parameter.
-            'InvalidRequestException',  # Invalid parameter value for the current state of the resource.
-            'ResourceNotFoundException',
-        ]:
-            PRINT('Encountered a known exception trying to acquire secrets.')
+        code = e.response['Error']['Code']
+        if code in _KNOWN_SECRETS_ERRORS:
+            logger.warning(f'Encountered a known exception ({code}) trying to acquire secrets.'
```

Comment:
I don't know why a `PRINT` was happening, but I figured it should be a log instead. I also am not sure why it needs to happen at all or why it doesn't show the actual error message, but I figured if it was going to hide the error message, then these explanatory comments would be good to change into visible messages. I'm slightly inclined to just show the actual error message... Thoughts from reviewers appreciated.